includes/privacy-functions.php

```diff
@@ -362,7 +362,7 @@ function _edd_anonymize_payment( $payment_id = 0 ) {
 	 *     @type string $message          A message to display if the customer could not be anonymized.
 	 * }
 	 */
-	$should_anonymize_payment = apply_filters( 'edd_should_anonymize_payment', array( 'should_anonymize' => true, 'message' => array() ), $payment );
+	$should_anonymize_payment = apply_filters( 'edd_should_anonymize_payment', array( 'should_anonymize' => true, 'message' => '' ), $payment );
 
 	if ( ! $should_anonymize_payment['should_anonymize'] ) {
```

Comment:
Since this variable is now filterable, this should use `empty()` instead of `!` to avoid debug notices.

Reply:
Good call.